snake/snake.hpp

```diff
@@ -13,6 +13,8 @@ class snake {
     unsigned short int snakeLength = 1;
     // saves the score
     unsigned short int score = 0;
+    // saves the highscore
+    unsigned short int highscores[10]{0};
```

Comment:
I think that this array should be moved into the highscore() function as it's only needed while parsing the scores file.